admin/views/tabs/network/restore-site.php

```diff
@@ -29,4 +32,6 @@
 }
 
 wp_nonce_field( 'wpseo-network-restore', 'restore_site_nonce', false );
-echo '<button type="submit" name="action" value="' . esc_attr( Yoast_Network_Admin::RESTORE_SITE_ACTION ) . '" class="button button-primary">' . esc_html__( 'Restore site to defaults', 'wordpress-seo' ) . '</button>';
+echo '<button type="submit" name="action" value="' . esc_attr( Yoast_Network_Admin::RESTORE_SITE_ACTION ) . '" class="button button yoast-button yoast-button--primary">' . esc_html__( 'Restore site to defaults', 'wordpress-seo' ) . '</button>';
```

Comment:
Class `button` should not be necessary. (and please note: you included it two times)

Reply:
Other buttons, e.g. "Save changes", do use the `button` class.
As far as I see, this class is the only one that adds a focus style to the red buttons.

"Save changes" button: normal and focused:

<img width="386" alt="chrome" src="https://user-images.githubusercontent.com/1682452/80479828-ed9afa80-894f-11ea-9b2c-e8b186fe9e9f.png">

"Restore..." button: normal, focused in Chrome, focused in Firefox:

<img width="751" alt="Screenshot 2020-04-28 at 12 51 34" src="https://user-images.githubusercontent.com/1682452/80479872-02778e00-8950-11ea-9a45-f34f2d23479e.png">

The blue outline is the Webkit native focus style. With Firefox, there's no visible focus style.

Also, the yellow buttons don't have any visible focus style.

I'd like to remind focus style is the only way keyboard users can actually see where they are while they navigate the user interface. A focus style must be provided, whether it's the one from the WordPress `button` class or our own style that should then be provided by the `yoast-button` class.

I do realise this problem is not specific to this PR and should go in a separate issue. /Cc @JoseeWouters